### PR TITLE
Add a support page

### PR DESCRIPTION
**Pull request by Anita-ihuman**
This PR is a fix for #381

### CODE REVIEW

**Comment by chipzoller** [comment]
Please use closing keywords or manually link to issues.

File: config/_default/menus/menu.en.toml [hunk 1]
```diff
@@ -25,3 +25,9 @@
   weight = -98
   url = "/community"
 
+
+[[main]]
+  name = "Supports"
```
Comment (chipzoller) [inline config/_default/menus/menu.en.toml:30]: Should just be "Support"

**Review by chipzoller — CHANGES_REQUESTED** [review]

**Review by chipzoller — CHANGES_REQUESTED** [review]
"Supports" => "Support" everywhere for consistency, including the path at `content/en/support/_index.md`

**Comment by Anita-ihuman** [comment]
@chipzoller please review

File: content/en/support/_index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+linkTitle: "Support"
+description: "Commercial Products and Services for Kyverno"
+type: docs
+---
+
+Below is a list of companies that offer commercial products and services for Kyverno.
+
+## Nirmata
+
+[Nirmata](https://nirmata.com/) is the creator of Kyverno and offers enterprise licensing with long term support, curated policy sets with compatibility testing, multi-cluster policy as code, a DevSecOps platform built with Kyverno, along with training and services for Kyverno.
```
Comment (chipzoller) [inline content/en/support/_index.md:12]: "[Nirmata] offers enterprise licensing..." <== is this actually true?

**Review by chipzoller — COMMENTED** [review]
I have asked Kim about this, but I don't see anywhere that Nirmata offers an enterprise license for Kyverno, only other offerings built around it.

**Comment by Anita-ihuman** [comment]
@JimBugwadia,
Please help me out with this.

**Comment by JimBugwadia** [comment]
The relevant offer is called "Nirmata Enterprise Subscription for Kyverno". See:

https://nirmata.com/kyverno-enterprise/

**Comment by chipzoller** [comment]
But that's not a "license" for Kyverno, that's mostly a support subscription entitlement, correct? Kyverno only has a single license and that's Apache 2.0 which includes commercial use. Just making sure we use the correct terminology here...

**Comment by JimBugwadia** [comment]
> But that's not a "license" for Kyverno, that's mostly a support subscription entitlement, correct? Kyverno only has a single license and that's Apache 2.0 which includes commercial use. Just making sure we use the correct terminology here...

I see your point. @Anita-ihuman - lets change to `and offers an enterprise subscription with long term support`.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**